src/content/tools/devtools/network.md

```diff
@@ -55,6 +55,20 @@ my-endpoint m:get t:json s:200
 https s:404
 ```
 
+### Recording network requests on app startup
+
+To record network traffic on app startup, you can start your app in a paused state then, begin recording before your app starts:
+
+1. Start your app in a paused state:
+    * `flutter run`
+      * use the `--start-paused` flag.
+    * `dart run`
+        * use the `--pause-isolates-on-start` flag.
+2. Connect DevTools to your paused app.
```

Comment:
"Open DevTools from the IDE where you started your app or from the link that was printed to command line if you started your app from the CLI."